libdnf/base/goal.cpp

```diff
@@ -108,15 +106,15 @@ void Goal::add_rpm_install(const libdnf::rpm::Package & rpm_package, bool strict
 
 void Goal::add_rpm_install(const libdnf::rpm::PackageSet & package_set, bool strict) {
     libdnf::rpm::solv::IdQueue ids;
-    for (auto package_id : *package_set.pImpl) {
+    for (auto package_id : *package_set.p_impl) {
         ids.push_back(package_id.id);
     }
     p_impl->install_rpm_ids.push_back(std::make_pair(std::move(ids), strict));
 }
 
 void Goal::add_rpm_install(const libdnf::rpm::SolvQuery & query, bool strict) {
```

Comment:
Minor - I suggest that the whole function could be removed

Reply:
Removed.